### ISSUE TITLE
Design visual aspect of website

### ISSUE THREAD

**Issue by sfiquet** [chingu-voyage4/Bears-Team-19#38]
Colours, fonts, text size, logo and other images, etc...

**Comment by sfiquet** [comment]
When it comes to implementation, here's how the React doc recommends creating a customised Bootstrap theme: create a package depending on Bootstrap, publish it with npm and add it as a dependency
https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#using-a-custom-theme

**Comment by Heyjp** [comment]
Since we have SCSS/SASS with the project now installed. I suggest we customize it via imports and the way it suggests on the bootstrap site. https://getbootstrap.com/docs/4.0/getting-started/theming/. Otherwise we might as well be using standard CSS which is what it seems to be using in the react docs.

**Comment by sfiquet** [comment]
Yes I was wondering about that. I just came across that info and thought it would be useful to keep a link to it to inform technical decisions. That doesn't mean we have to do it like that.
